packages/app/src/systems/Core/utils/math.ts

```diff
@@ -0,0 +1,17 @@
+import type { BNInput } from 'fuels';
+import { bn } from 'fuels';
+
+import { DECIMAL_UNITS, MAX_FRACTION_DIGITS } from '~/config';
+
+export function formatAmount(amount?: BNInput) {
+  return bn(amount).format({
+    minPrecision: MAX_FRACTION_DIGITS,
+  });
+}
+
```

Comment:
with new changes of [ts-sdk](https://github.com/FuelLabs/fuels-ts/pull/624), `minPrecision` will already be `3` 

 I guess the change needed is in `precision` now:
```
return bn(amount).format({
    precision: MAX_FRACTION_DIGITS,
});
```